src/main/kotlin/lotto/view/ResultView.kt

```diff
@@ -0,0 +1,29 @@
+package lotto.view
+
+import lotto.domain.Lotto
+
+object ResultView {
+
+    fun showLottoNumber(lottoNumbers: MutableList<List<Number>>) {
+        lottoNumbers.forEach { println(it) }
+    }
+
+    fun showPrizeStatics(prizes: List<Lotto>, profitRate: Double) {
+        println(
+            "당첨 통계\n" + "---------"
+        )
+        prizes.forEach { it ->
```

Comment:
아래의 코드로 개선해 보면 어떨까요?

```suggestion
        prizes.forEach {
```